overlay.d/05core/usr/lib/dracut/modules.d/40ignition-ostree/coreos-relabel

```
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -euo pipefail
+
+err() {
+    echo "$@" >&2
+}
+
+fatal() {
+    err "$@"
+    exit 1
+}
+
+if [ $# -eq 0 ]; then
+    err "Usage: $0 [PATTERN...]"
+    err " e.g.: $0 /etc/passwd '/etc/group*'"
+fi
+
+source /sysroot/etc/selinux/config
```

Maybe silently do nothing if that file doesn't exist, to be nicer to people who want to build custom systems without SELinux?  There's a similar pattern with other tools like `restorecon`.